src/App.css

```diff
@@ -0,0 +1,43 @@
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style-type: none;
+}
+
+.desktop-friendly {
+  max-width: 534px;
+  margin: 0 auto;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+}
+
+.grid-item {
+  padding: 5px 15px 50px 15px;
+  color: white;
+  border: 1px solid #eee;
+  background-color: #634e46ff;
+}
+
+.grid-item:nth-child(odd) {
+  background-color: #d61a3c;
+}
+
+.color-2 {
+  background-color: #634e46ff;
+}
```

Comment:
- [ ] You should kindly select one main color for the app and be consistent (it does **not** need to be the same color as in the design guidelines). I think you have two different colors on your App